boundary_layer_default_plugin/config/operators/dataflow_template.yaml

```diff
@@ -33,6 +33,7 @@ parameters_jsonschema:
                     type: string
                 maxWorkers:
                     type: string
+                    default: '1000'
```

Comment:
To me this seems like it does not belong here.  This is putting Etsy-specific constraints into a utility that is meant to be more of a generic pass-through of information from configurations to airflow DAGs (and that might be in use by other people who might not agree that this is the proper default value).

I think if you want to set this default, there are two better places to do it:
1. In the source code of [airflow](https://github.com/apache/airflow) itself (if the maintainers of that would approve this...).  This way the business logic exists inside airflow, not inside boundary-layer
2. In the Etsy-customized boundary-layer plugin.  It might require that you copy this entire config file into there, but that's OK I think.

Reply:
Thanks Kevin, that makes sense, i was also a bit skeptic when i could not find many defaults being set in this repo. 
I think following your second approach will workout best for us, because first one might need an airflow deploy, let me look more into the best place for it.